app/src/main/java/com/github/epfl/meili/LogoActivity.kt

```diff
@@ -0,0 +1,40 @@
+package com.github.epfl.meili
+
+import android.content.Intent
+import android.os.Bundle
+import android.os.Handler
+import androidx.appcompat.app.AppCompatActivity
+import com.github.epfl.meili.home.Auth
+import com.github.epfl.meili.home.FirebaseAuthenticationService
+import com.github.epfl.meili.home.GoogleSignInActivity
+import com.github.epfl.meili.map.MapActivity
+import com.github.epfl.meili.util.UserPreferences
+
+class LogoActivity : AppCompatActivity() {
+    companion object {
+        private const val LOGO_DISPLAY_TIME = 500L // milliseconds
```

Comment:
In my opinion, we should either increase this time or remove the app slogan from the starting screen. Since we know what the slogan says we probably read it faster than the average user (especially since revolutionised is such a complicated word).

Reply:
Sure, I don't really have a strong opinion about this to be honest, let's just ask on the group to see what everyone thinks.

Reply:
The group agrees on 1 second!